src/geovista/transform.py

```diff
@@ -201,16 +202,16 @@ def transform_point(
         :meth:`pyproj.crs.CRS.from_user_input`.
     x : ArrayLike
         The spatial point x-value, in canonical `src_crs` units, to be
-        transformed from the `src_crs` to the `tgt_crs`. May be scalar
-        or 1-D.
+        transformed from the `src_crs` to the `tgt_crs`. Must be scalar
+        (0-dimensional).
```

Comment:
@trexfeathers It's possible to pass a scalar, a 0-D array or a 1-D array with a single value.

All will work here 👍

Reply:
I don't think you and I are using the same definition for dimensionality. I was going from what I get from `numpy.array.ndim`, which is `1` for all of the following:

```
np.array([]).ndim
np.array([1]).ndim
np.array([1, 2]).ndim
```

Can we arrive at some phrasing which means the same thing to everyone?

Reply:
For me a scalar array is 0-D e.g., `np.array(1)`, but I've used the term loosely such that scalar also covers a single value e.g., `1`.

As you highlight above, those are all examples of 1-D arrays.

I think the only clarification that's required here is something like:

`May be a scalar or a single valued 1-D array`

Reply:
e0e75e8ddc8f283495874f79ecf3623790bded64